requirements-training.txt

```diff
@@ -4,6 +4,6 @@ h5py
 numpy >= 1.16.6
```

Comment:
Where is this file used?

Reply:
https://github.com/microsoft/onnxruntime/blob/123e1eac019b1c25da6819254f962cc08ddb060d/setup.py#L509

So, when the pipeline does `pip install *.whl`, it will try to first install all the dependencies listed in setup.py.